### ISSUE TITLE
gutcheck which fields are used for v3 data call

### ISSUE THREAD

**Issue by gbinal** [GSA/site-scanning#1110]
- 'scan_date',
- 'primary_scan_status',
- 'accessibility_scan_status',
- 'performance_scan_status',
- 'security_scan_status',
- 'robots_txt_scan_status',
- 'target_url_agency_owner',
- 'target_url_bureau_owner',
- 'target_url',
- 'target_url_domain',
- 'target_url_top_level_domain',
- 'dap',
- 'accessibility_violations',
- 'viewport_meta_tag',
- 'largest_contentful_paint',
- 'cumulative_layout_shift',
- 'https_enforced',
- 'hsts',
- ‘third_party_service_domains’,
- 'uswds_usa_class_list',
- 'uswds_banner_heres_how',
- 'uswds_semantic_version',
- 'title',
- 'description',
- 'sitemap_xml_detected',
- 'robots_txt_sitemap_locations',
- 'robots_txt_detected',
- 'og_title',
- 'og_description',
- 'og_url',
- 'site_search',
- 'cms',
- ‘required_links_url’,

‘required_links_text’
